### ISSUE TITLE
Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll.

### ISSUE THREAD

**Issue by EvilMudkip** [tensorflow/tensorflow#13319]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug or a feature request.
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No.
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows 10.
- **TensorFlow installed from (source or binary)**: Binary.
- **TensorFlow version (use command below)**: tensorflow-1.3.0-cp36-cp36m-win_amd64.whl
- **Python version**: 3.6.
- **Bazel version (if compiling from source)**: N/A.
- **CUDA/cuDNN version**: N/A.
- **GPU model and memory**: GT 650M.
- **Exact command to reproduce**: `python` followed by `import tensorflow as tf`.

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

I followed exactly on how to install Tensorflow, but whenever I try to do something involving Tnesorflow this error comes up. I installed Tensorflow using Anaconda 4.4. I first typed `conda create -n tensorflow python=3.6` then `activate tensorflow` then `pip install --ignore-installed --upgrade tensorflow`. After that, I typed `python` then `import tensorflow as tf` I get this error: `Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll.`
![untitled](https://user-images.githubusercontent.com/31815007/30871645-1adda5ac-a31a-11e7-8ec8-8a323e8054a8.png)

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by poxvoculi** [comment]
The installation guide includes this warning:
 "The Anaconda installation is community supported, not officially supported."

You might try the pip install, or see whether you can get help with the conda install on stackoverflow.

**Comment by gunan** [comment]
Also, we have no MKL support on Windows yet.
All our build rules are only written for ubuntu at the moment.

**Comment by bartybooy** [comment]
I had 2 versions of Python installed. One basic version of Python and one installed by Anaconda. When running my machine learning script it threw the same error. When I uninstalled one of the two versions (in my case I removed Anaconda) everything works fine again. I use deeplearning4j as library and apparently it has difficulties to resolve this dll, cause there were multiple Python installations installed.
